src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -161,6 +163,14 @@ define([
             }
         },
 
+        _executeEvent: function() {
```

Comment:
i suggest if you add `doNothing` into the widge events add `if(this.callEvent === "doNothing") { return; }`

Reply:
Widget requires a microflow or nanoflow at all times since that's its major functionality.